control-plane/pkg/reconciler/channel/controller.go

```diff
@@ -89,10 +90,12 @@ func NewController(ctx context.Context, watcher configmap.Watcher, configs *conf
 		)
 	}
 
```

Comment:
You'll need to create `features` up here before you get the CA certs so that you are able to validate if you should have an error or not:
https://github.com/knative-sandbox/eventing-kafka-broker/blob/32dfed583ceb02c4fa1581b3bffab07d6ec0e7d6/control-plane/pkg/reconciler/broker/controller.go#L98